import/marc_local.properties

```diff
@@ -87,3 +87,14 @@
 # library or repository systems and can be a useful match point with third party
 # systems.
 #uuid_str_mv = 024$a ? (ind1 == 7 && ($2 == 'uuid' || $2 == 'UUID')
+
+# Uncomment the following lines to use SolrMarc plug-in for indexing MARC-based hierarchies
+#hierarchytype = custom(org.vufind.Collection), getHierarchyType
```

Comment:
Should the class be org.vufind.index.Collection, based on the package specified in the Java file?